fs/copy.py

```diff
@@ -54,100 +53,168 @@ def copy_fs_if_newer(
     workers=0,  # type: int
 ):
     # type: (...) -> None
-    """Copy the contents of one filesystem to another, checking times.
+    """Use ``copy_fs_if`` instead."""
+    warnings.warn(
+        DeprecationWarning("copy_fs_if_newer is deprecated. Use copy_fs_if instead.")
+    )
```

Comment:
I think right way to raise warnings is (according to the stdlib [docs](https://docs.python.org/3/library/warnings.html)):
```python
warnings.warn("copy_fs_if_newer id deprecated. Use copy_fs_if instead", DeprecationWarning)
```